src/main/java/net/minecraftforge/common/data/ForgeItemTagsProvider.java

```diff
@@ -141,6 +141,70 @@ public void registerTags()
         copy(Tags.Blocks.STORAGE_BLOCKS_QUARTZ, Tags.Items.STORAGE_BLOCKS_QUARTZ);
         copy(Tags.Blocks.STORAGE_BLOCKS_REDSTONE, Tags.Items.STORAGE_BLOCKS_REDSTONE);
         getBuilder(Tags.Items.STRING).add(Items.STRING);
+        getBuilder(Tags.Items.WEAPONS).add(Tags.Items.SWORDS, Tags.Items.BOWS, Tags.Items.CROSSBOWS, Tags.Items.TRIDENTS);
+        getBuilder(Tags.Items.SWORDS).add(Tags.Items.SWORDS_WOODEN, Tags.Items.SWORDS_STONE, Tags.Items.SWORDS_IRON, Tags.Items.SWORDS_GOLDEN, Tags.Items.SWORDS_DIAMOND);
+        getBuilder(Tags.Items.SWORDS_WOODEN).add(Items.WOODEN_SWORD);
+        getBuilder(Tags.Items.SWORDS_STONE).add(Items.STONE_SWORD);
+        getBuilder(Tags.Items.SWORDS_IRON).add(Items.IRON_SWORD);
+        getBuilder(Tags.Items.SWORDS_GOLDEN).add(Items.GOLDEN_SWORD);
+        getBuilder(Tags.Items.SWORDS_DIAMOND).add(Items.DIAMOND_SWORD);
+        getBuilder(Tags.Items.BOWS).add(Items.BOW);
+        getBuilder(Tags.Items.CROSSBOWS).add(Items.CROSSBOW);
+        getBuilder(Tags.Items.TRIDENTS).add(Items.TRIDENT);
+        
+        getBuilder(Tags.Items.TOOLS).add(Tags.Items.HOES, Tags.Items.AXES, Tags.Items.PICKAXES, Tags.Items.SHOVELS);
```

Comment:
This does not include the list of shears, even though they are in the sub directory/tag for tools.

Reply:
Yes, I just noticed, on it ^^